source/code/java/authentication/mfa_get_account_factors1_resp.java

```diff
@@ -0,0 +1,12 @@
+// In order to get all Factors
+FactorList factors = account.getFactors();
+
+Map<String,String> parameters = new HashMap<>(1);
+
+// In order to get SmsFactor
+parameters.put("type", "sms");
```

Comment:
Please use `FactorCriteria` instead of the Map as it is type-safe and simpler to read.